src/Servers/Kestrel/Core/ref/Microsoft.AspNetCore.Server.Kestrel.Core.netcoreapp3.0.cs

```diff
@@ -840,6 +841,7 @@ public partial class HttpResponsePipeWriter : System.IO.Pipelines.PipeWriter
     public partial class HttpResponseTrailers : Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.HttpHeaders
     {
         public HttpResponseTrailers() { }
+        public Microsoft.Extensions.Primitives.StringValues HeaderContentLength { get { throw null; } set { } }
```

Comment:
Do we actually use/need HttpResponseTrailers.ContentLength? Why is that and ThrowInvalidContentLengthException being added?

Reply:
Removed